nixos/modules/services/monitoring/netdata.nix

```diff
@@ -158,11 +158,28 @@ in {
         # User and group
         User = cfg.user;
         Group = cfg.group;
-        # Runtime directory and mode
-        RuntimeDirectory = "netdata";
-        RuntimeDirectoryMode = "0755";
         # Performance
         LimitNOFILE = "30000";
+        # Runtime directory and mode
+        RuntimeDirectory = "netdata";
+        RuntimeDirectoryMode = "0750";
+        # State directory and mode
+        StateDirectory = "netdata";
```

Comment:
There are some tmpfiles rules above that should be deleted as they duplicate this.

Reply:
Fixed.